{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "f5c1e25f_c471eadf",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 3
      },
      "lineNbr": 0,
      "author": {
        "id": 5634
      },
      "writtenOn": "2024-04-23T11:37:43Z",
      "side": 1,
      "message": "I think this is ready for review - internal bot is failing due to a mock; I have a CL out to fix that.",
      "revId": "7073cadf491ade81d41f5a11ab753d7f96363f07",
      "serverId": "58829da1-049c-39fb-b951-ebdcd0984767"
    }
  ]
}